app/javascript/controllers/sortable_list_controller.js

```diff
@@ -0,0 +1,27 @@
+import { Controller } from "@hotwired/stimulus"
+import { Sortable } from "sortablejs";
+
+export default class extends Controller {
+  static values = {
+    groupName: String,
+    scrollSensitivity: {
+      type: Number,
+      default: 50,
+    },
+    scrollSpeed: {
+      type: Number,
+      default: 8,
+    }
+  }
+  connect() {
+    this.sortable = new Sortable(this.element, {
+      scroll: true,
+      forceFallback: true,
```

Comment:
Can you remove this forceFallback as its causing some weird behaviour where the elements the list item that is selected is hovered over get highlighted. Removing this fixes the issue

Reply:
Removed in eed8e25f89304aabec0116cd5d92acc7eab71a95